CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/ValueInputPane.java

```diff
@@ -294,18 +294,21 @@ protected void updateItem(final String item, final boolean empty) {
     @Override
     public void recentValuesChanged(final RecentValuesChangeEvent e) {
         if (recentValuesCombo != null && parameterId.equals(e.getId())) {
-            recentValuesCombo.getSelectionModel().selectedIndexProperty().removeListener(recentValueSelectionListener);
-            final List<String> recentValues = e.getNewValues();
-            if (recentValues != null) {
-                recentValuesCombo.setItems(FXCollections.observableList(recentValues));
-                recentValuesCombo.setDisable(false);
-            } else {
-                final List<String> empty = Collections.emptyList();
-                recentValuesCombo.setItems(FXCollections.observableList(empty));
-                recentValuesCombo.setDisable(true);
-            }
-            recentValuesCombo.setPromptText("...");
-            recentValuesCombo.getSelectionModel().selectedIndexProperty().addListener(recentValueSelectionListener);
+            //#1608: JavaFX code allows updating the UI from an JavaFX application thread.
+             Platform.runLater(() -> {             
```

Comment:
Seems this has an extra space at the front (relative to column I think it is meant to sit in). Might possibly apply to the whole code block(?)